pkg/webhook/webhook.go

```diff
@@ -34,6 +34,11 @@ var (
 	podLabelMissingWarning = fmt.Sprintf(`pod missing label '%s: "true"'. This label will be required in a future release for the webhook to mutate pod. Please add the label to the pod.`, UseWorkloadIdentityLabel)
 )
 
+const (
+	// warningAnnotationKey is the annotation key used to store warning messages in audit annotations
+	warningAnnotationKey = "warning"
```

Comment:
Check the docs of the `AuditAnnotations` field - these keys have to be prefixed based on the webhook.

Reply:
Ah I misread https://github.com/kubernetes/kubernetes/blob/17bf864c1fc20e4badc9a6e0659c5190310462e1/pkg/apis/admission/types.go#L135-L140 and thought it's added after the response is returned. Let me modify it.

Reply:
done!